Lib/test/test_uuid.py

```diff
@@ -1140,6 +1140,23 @@ def test_uuid_weakref(self):
         weak = weakref.ref(strong)
         self.assertIs(strong, weak())
 
+
+class TestUUIDCli(BaseTestUUID, unittest.TestCase):
```

Comment:
Why are we inheriting from `BaseTestUUID`? we don't use anything from it and we don't want the tests to be duplicated.